### ISSUE TITLE
Error when running SCtransform

### ISSUE THREAD

**Issue by kelvinlee760948065** [ChristophH/sctransform#117]
![b11adb6e4d41b7e8c4008e3e519eba2](https://user-images.githubusercontent.com/71078929/143983391-d08a4f2f-e56d-4e1a-8414-97f2b6b25d20.png)

**Comment by ChristophH** [comment]
Yes, looks like there was an error. I'm going to need more information than that screenshot to identify the problem. Can you provide a [minimal reproducible example](https://www.dummies.com/programming/r/how-to-make-a-minimal-reproducible-example-to-get-help-with-r/)?
